features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/AppRouts.jsx

```diff
@@ -14,35 +14,35 @@
  * limitations under the License.
  */
 
-import React from 'react';
-import { Redirect, Route, Switch } from 'react-router-dom';
-import { withStyles } from '@material-ui/core';
-import TagCloudListing from 'AppComponents/Apis/Listing/TagCloudListing';
-import Apis from 'AppComponents/Apis/Apis';
-import Landing from 'AppComponents/LandingPage/Landing';
-import ApplicationFormHandler from 'AppComponents/Applications/ApplicationFormHandler';
-import { PageNotFound, ScopeNotFound } from 'AppComponents/Base/Errors';
-import RedirectToLogin from 'AppComponents/Login/RedirectToLogin';
-import SettingsBase from 'AppComponents/Settings/SettingsBase';
-import Listing from 'AppComponents/Applications/Listing/Listing';
-import Details from 'AppComponents/Applications/Details/index';
+import React from "react";
```

Comment:
shall we fix these eslint issues? double quote issues and there are few eslint spacing issue below too

Reply:
Yes. Having trouble with auto-fix in vscode. Will fix them.

Reply:
Fixed.